Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -179,6 +185,7 @@ public struct Configuration: Equatable {
             }
             self.init(dict: dict)!
             configurationPath = fullPath
+            hash = yamlContents.hash
```

Comment:
instead of hashing the yaml contents, we could hash the properties of this struct, meaning that changes to the configuration file like reordering items or adding comments wouldn't invalidate the cache.